turbopy/core.py

```diff
@@ -337,6 +337,14 @@ def is_running(self):
 
 
 class Grid:
+    """Grid class
```

Comment:
I added placeholder docstrings for Grid and grid_data so your additions would show up when the documentation is built. If there is no class level docstring, Sphinx will not use the lower level docstrings.  Also, I added a placeholder for the grid_data attribute to demonstrate cross referencing the object in lower level documentation.

Reply:
Thanks! This made me confused for a while on a different project, until I realized that Sphinx needed the class level docstring.